test/validation/api/classification/odp_classification_common.c

```diff
@@ -104,7 +104,10 @@ int cls_pkt_set_seq(odp_packet_t pkt)
 	offset = odp_packet_l4_offset(pkt);
 	CU_ASSERT_FATAL(offset != ODP_PACKET_OFFSET_INVALID);
 
-	if (ip->proto == ODPH_IPPROTO_UDP)
+	if (ip->proto == ODPH_IPPROTO_SCTP) {
+		status = odp_packet_copy_from_mem(pkt, offset + ODPH_SCTPHDR_LEN,
```

Comment:
This puts the seqno to where the first chunk of the SCTP packets should be, resulting in an invalid chunk (in what way, depends on endianess and the value of the seqno). I am not sure if it is ok to use badly formatted SCTP packets in this test.

I would suggest either creating a valid chunk (maybe data chunk) and placing the seqno differently, or adding a big bold comment that we are intentionally using a somewhat invalid SCTP packet for this testing under the assumption that no implementation really cares.

Reply:
Ack, Will add comment.